harvester_e2e_tests/integrations/test_9_rancher_integration.py

```diff
@@ -130,7 +130,14 @@ def harvester_mgmt_cluster(api_client, rancher_api_client, unique_name, polling_
 @pytest.fixture(scope='module')
 def harvester_cloud_credential(api_client, rancher_api_client,
                                harvester_mgmt_cluster, unique_name):
-    harvester_kubeconfig = api_client.generate_kubeconfig()
+    code, data = rancher_api_client.clusters.generate_kubeconfig(
```

Comment:
I think the two configurations are same, could you check with @albinsun about the change?

Reply:
They are not the same. One uses Rancher's API endpoint and authenticates with a Rancher token and the other one uses Harvester's API endpoint directly.
For these tests we need to use Rancher's API endpoint.
Please read the issue linked up top.